server/api/settings.py

```diff
@@ -40,6 +40,7 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
+    "teams"
```

Comment:
the teams folder should be under api -> api.team.

the folder should also be "team" not "teams"